pkgs/development/php-packages/phpcbf/default.nix

```diff
@@ -1,35 +1,27 @@
-{ mkDerivation, fetchurl, lib, php, makeWrapper }:
+{ stdenv
```

Comment:
File can be removed, it is now dead code.

Reply:
Fixed.